static/app/views/issueDetails/traceTimeline/traceTimeline.spec.tsx

```diff
@@ -161,34 +162,38 @@ describe('TraceTimeline', () => {
     });
     MockApiClient.addMockResponse({
       url: `/organizations/${organization.slug}/events/`,
-      body: discoverBody,
+      body: emptyBody,
       match: [MockApiClient.matchQuery({dataset: 'discover'})],
     });
     // I believe the call to projects is to determine what projects a user belongs to
     MockApiClient.addMockResponse({
       url: `/organizations/${organization.slug}/projects/`,
       body: [],
     });
+    // We need the metadata from the issue in order to render the related issue
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/issues/1000/`,
```

Comment:
I would love to avoid an extra request here if possible to avoid adding more latency. You might be able to add the `message` field to the discover `/events/` call? If so you should have all the necessary information to render this new design.

Reply:
Maybe I can include `groupMetadata` per event. That way we can even fix events in the trace timeline and the new trace view.